core/trashcan.js

```diff
@@ -426,8 +425,8 @@ Blockly.Trashcan.prototype.openFlyout = function() {
   if (this.contentsIsOpen()) {
     return;
   }
-  var xml = this.contents_.map(Blockly.Xml.textToDom);
-  this.flyout.show(xml);
+  var contents = this.contents_.map((str) => JSON.parse(str));
```

Comment:
Arrow functions are not allowed in ie11. https://caniuse.com/?search=arrow%20functions